chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -9,6 +9,8 @@
       "access_keys": "",
       "shell_prompt": "USER@HOST:CWD SUFFIX",
       "enable_pxe": true,
+      "self_update": false,
+      "self_update_url": "http://updates.suse.com/sle12/$arch",
```

Comment:
Also, I'm not convinced we need a boolean to enable this or not. Either the repo is present locally (we enable), or the user provides an external URL (we enable), or the repo is not present locally and there's no url (we disable)